seqr/views/utils/airflow_utils.py

```diff
@@ -1,30 +1,52 @@
+from collections import defaultdict, OrderedDict
+from datetime import datetime
+from django.contrib.auth.models import User
+from django.db.models import F
+import itertools
 import json
 from google.auth.transport.requests import Request
 from google.oauth2 import id_token
+import re
 import requests
 
+from reference_data.models import GENOME_VERSION_GRCh38, GENOME_VERSION_LOOKUP
+from seqr.models import Individual, Sample, Project
 from seqr.utils.communication_utils import safe_post_to_slack
+from seqr.utils.file_utils import get_gs_file_list, does_file_exist
 from seqr.utils.logging_utils import SeqrLogger
+from seqr.utils.search.utils import backend_specific_call
+from seqr.views.utils.export_utils import write_multiple_files_to_gs
 from settings import AIRFLOW_API_AUDIENCE, AIRFLOW_WEBSERVER_URL, AIRFLOW_DAG_VERSION, \
     SEQR_SLACK_LOADING_NOTIFICATION_CHANNEL
 
 logger = SeqrLogger(__name__)
 
+SEQR_DATASETS_GS_PATH = 'gs://seqr-datasets/v02'
+
 
 class DagRunningException(Exception):
     pass
 
 
-def trigger_data_loading(dag_name, projects, data_path, additional_dag_variables, user,
-                         success_message, success_slack_channel, error_message):
+def trigger_data_loading(projects: list[Project], sample_type: str, data_path: str, user: User, success_message: str,
+                         success_slack_channel: str, error_message: str,
+                         dataset_type: str = Sample.DATASET_TYPE_VARIANT_CALLS, genome_version: str = GENOME_VERSION_GRCh38,
```

Comment:
This is maybe me being paranoid about lack of type safety, but is there a strong reason for us to default to `SNV_INDEL` here rather than force the caller to supply a dataset_type?

Reply:
no strong reason, I can make this explicit